labs-ios-starterTests/NetworkServiceTests.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  NetworkServiceTests.swift
+//  labs-ios-starterTests
+//
+//  Created by Kenny on 9/9/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import XCTest
+@testable import labs_ios_starter
+
+class NetworkServiceTests: XCTestCase {
+    let networkService = NetworkService()
+
+    func testGetRequestNoError() {
+        let expectation = self.expectation(description: "Test GET Request Valid URL No Error")
```

Comment:
could also do `XCTestExpectation(` if you liked how that looks better than `self.expectation(`